typescript-data-model/order-history.ts

```diff
@@ -0,0 +1,76 @@
+interface item {
+  itemName: string;
+  author?: string;
+  price: number;
+  review: string[];
+}
+interface order {
+  'order placed': Date;
+  total: number;
+  'ship to': string;
+  'order number': string;
+  items: item[];
+}
+
+const order01: order = {
+  'order placed': new Date('2020-08-04'),
+  total: 34.0,
+  'ship to': 'JS Masher',
+  'order number': '114-999999999',
```

Comment:
Property names should not have spaces in them. It works because you made them strings, but it's a bad practice. Property names follow the same naming conventions as variables, and should be using `camelCased` instead.

So this should be `orderPlaced`, `shipTo`, `orderNumber`, etc